tests/fixtures/pddl_files/storage/domain.pddl

```diff
@@ -0,0 +1,48 @@
+; IPC5 Domain: Storage Propositional
+; Authors: Alfonso Gerevini and Alessandro Saetti
+
+(define (domain Storage-Propositional)
+(:requirements :typing)
+(:types hoist surface place - object
```

Comment:
Commit 22643a2bd9bd41c5ab120ccf48c5f1fae47485a4 changed this line. Is that a right change?

Reply:
Reason: area already stated below.